src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data.py

```diff
@@ -794,14 +789,42 @@ def migrate_notices(bind):
     link_notice_history_to_translation_id = text(
         """
         UPDATE privacynoticehistory
-        SET translation_id = noticetranslation.id, title = privacynoticehistory.name, language = 'en'
+        SET translation_id = noticetranslation.id, title = privacynoticehistory.name
         FROM noticetranslation
         WHERE noticetranslation.privacy_notice_id = privacynoticehistory.privacy_notice_id;
         """
     )
 
     bind.execute(link_notice_history_to_translation_id)
 
+    # Queries the latest historical version for each notice
+    new_notice_histories_to_create_query = text(
+        """
+        select privacy_notice_id, max(version) from privacynoticehistory group by privacy_notice_id;
+        """
+    )
+    # Create a new privacy notice history record that bumps the version and clears regions and displayed_in* fields and sets language
+    for res in bind.execute(new_notice_histories_to_create_query):
+        create_privacy_notice_history_query = text(
+            """
+            INSERT INTO privacynoticehistory (id, name, description, origin, consent_mechanism, data_uses, version, disabled, enforcement_level, has_gpc_flag, internal_description, notice_key, gpp_field_mapping, framework, language, title, translation_id, privacy_notice_id)
+            SELECT :record_id, name, description, origin, consent_mechanism, data_uses, :new_version, disabled, enforcement_level, has_gpc_flag, internal_description, notice_key, gpp_field_mapping, framework, :language, title, translation_id, privacy_notice_id
+            FROM privacynoticehistory
+            WHERE version = :current_version AND
+            privacy_notice_id = :privacy_notice_id
+        """
+        )
+        bind.execute(
+            create_privacy_notice_history_query,
+            {
+                "record_id": generate_record_id("pri"),
+                "language": "en",
+                "new_version": res["max"] + 1,
+                "current_version": res["max"],
+                "privacy_notice_id": res["privacy_notice_id"],
+            },
+        )
+
```

Comment:
We're no longer removing "regions" and displayed_in* columns from the historical notice tables for auditing purposes for notices created and displayed to end users before this migration.  

However, it would be confusing to still have these values on the latest historical records against which we're continuing to save preferences.  So I now create a new historical record for each privacy notice and bump the version.  Regions, and displayed_in* columns are not copied to the new version.